Octokit/Models/Request/RepositoryTransfer.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryTransfer
```

Comment:
XmlDoc comments should be added to this class including it's ctor/s and the public properties